bolt/lib/Rewrite/LinuxKernelRewriter.cpp

```diff
@@ -593,20 +631,26 @@ Error LinuxKernelRewriter::processORCPostCFG() {
           continue;
         }
 
-        // In case there was no ORC entry that matched the function start
-        // address, we need to propagate ORC state from the previous entry.
+        // Get state for the start of the function.
         if (!CurrentState) {
           auto It =
               llvm::partition_point(ORCEntries, [&](const ORCListEntry &E) {
-                return E.IP < BF.getAddress();
+                return E.IP <= BF.getAddress();
```

Comment:
Why `<=` ? Can you have overlapping addresses?

Reply:
We are finding the first address larger than the function start. Yes, a terminator entry can have the same address as the function start, but should precede it.

Reply:
Makes sense, can you just add a comment explaining this?